### DISCUSSION TITLE
Global & Environment Variables - Reference Variables are not evaluated in the Report

### DISCUSSION THREAD

**Discussion by prasanmgc** [DannyDainton/newman-reporter-htmlextra#430] [Ideas]
@DannyDainton Thanks for the wonderful report. I noticed that when the Global & Environment Variables that contain Reference Variables (i.e. Concatenation of 2 other variables), report is not showing the evaluated value for the Reference Variable. Instead, it shows the same names as given in the Postman. Are there any plans in implementing the same?

Example:
Global/Environment Variable:
First Name               XXXXX
Last Name                YYYYY
Full Name                {{First Name}} {{Last Name}}

Currently the report is displaying Full Name as {{First Name}} {{Last Name}} instead of XXXXX YYYYY

**Comment by DannyDainton** [comment]
It's nothing to do with the reporter if those are not rendering correctly. That's what's coming through from the Newman Summary Object - That's where the report data comes from.

Do you see the same thing if you only use the CLI reporter?

I don't know anything about your context or how your Collection/Environment looks or what the command was that you ran this with.

If you have exported the environment file, ensure that the values are in the initial value field because if those are only in the current value...They wouldn't have been included in the export file.
